src/app/components/shared/annotation-download/annotation-download.component.spec.ts

```diff
@@ -158,7 +158,7 @@ describe("AnnotationDownloadComponent", () => {
     it("should default to UTC if no timezone exists for site", () => {
       setup([projectKey, siteKey], [defaultProject, siteWithoutTimezone]);
       spec.detectChanges();
-      expect(spec.component.model.timezone).toBe("UTC");
+      expect(spec.component.model.timezone).toBe("Etc/UTC");
```

Comment:
make sure there is a test elsewhere in this file that tests the presentation value of the timezone remains unchanged.

We don't use the slash style timezones in the presented text, so hopefully this change won't even show to a user.

also make sure that the network request to the server is well understood. Did you try downloading an annotation files with `Etc/UTC` is the HTTP request parameters?

Reply:
Presentation tests: Complete

`Etc/UTC` is sent in requests when downloading annotation files